### DISCUSSION TITLE
Filament RFID JSON Dump?

### DISCUSSION THREAD

**Discussion by FroggMaster** [raleighlittles/Bambu_Lab_Utility_Scripts#3] [General]
I'm curious if you'd be willing to share the discord where you're obtaining BIN/JSON dump files for Filaments? I'd be happy to provide some myself. Would also love to download the collection of JSON files if possible. :)

**Comment by FroggMaster** [comment]
I was able to locate the Discord Community after reaching out to another user.

**Comment by raleighlittles** [comment]
Here you go! https://discord.gg/YnxBkTxK

I have a private repository that contains the bin/json filament dumps. I wanted to actually ask some of the other Discord members if they would be okay with me sharing that repository publicly first.
